build/fpm/src/rpm/config.single.el9.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+pkg_name="php-relay"
+pkg_provides="php-relay"
+pkg_identifier="el9"
+
+pkg_binary_dest=(
+    "usr/lib64/php/modules"
+    "usr/lib64/php-zts/modules"
+)
+
+pkg_config_dest=(
+    "etc/php.d/60-relay.ini"
+    "etc/php-zts.d/60-relay.ini"
+)
```

Comment:
Thanks, the `php82-php-pecl-redis5.el9.remi.x86_64` was very helpful. Can you post one of Remi's "single" package as well so we can verify these haven't changed?